app/views/payments/_payment.html.haml

```diff
@@ -0,0 +1,34 @@
+= semantic_form_for(@payment, url: conference_payments_path) do |f|
+  .form-group
+    = f.label :first_name, 'First name(as on card)'
```

Comment:
I think we could just ask for full name, instead of first & last, and call the text field `:name`, which should be allowed by the credit card class:
https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/credit_card.rb#L251
That would simplify this form, because really there's no reason to ask for both separately. What do you think>